frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/form/form-field-row.tsx

```diff
@@ -23,22 +25,47 @@ export const FormFieldRow: React.FC<FieldFormRowProps> = ({
     return null;
   }
 
+  const value = iGetFieldValue(field);
   const loading = isLoading(loadingResources);
 
   return (
-    <FormRow
-      fieldId={getFieldId(fieldKey)}
-      title={fieldType === FormFieldType.INLINE_CHECKBOX ? undefined : getFieldTitle(fieldKey)}
-      help={getFieldHelp(fieldKey, iGet(field, 'value'))}
-      isRequired={isFieldRequired(field)}
-      validationMessage={iGetIn(field, ['validation', 'message'])}
-      validationType={iGetIn(field, ['validation', 'type'])}
-      isLoading={loading}
-    >
-      <FormFieldContext.Provider value={{ field, fieldType, isLoading: loading }}>
-        {children}
-      </FormFieldContext.Provider>
-    </FormRow>
+    <FormFieldReviewContext.Consumer>
+      {({ isReview }: { isReview: boolean }) => {
+        if (isReview) {
```

Comment:
nit: for better readability, I would implement these two branches as two components.

Reply:
+1 I created `FormFieldReviewRow` component but left the other branch - because it is a single component and another wrapping layer would not help that much